app/src/main/java/com/hamdan/forzenbook/data/network/LoginService.kt

```diff
@@ -4,9 +4,11 @@ import retrofit2.Response
 import retrofit2.http.Body
 import retrofit2.http.GET
 import retrofit2.http.POST
+import retrofit2.http.Query
 
 interface LoginService {
-    @POST("api/login")
```

Comment:
You still need a POST endpoint. Where is it?

Reply:
Thats the other ticket for login isn't it? I thought it was out of scope so I didn't put it but I can include it if thats better.

Reply:
Sure that's fine, you can leave a TODO if you want

Reply:
ok will do